phpunit.xml.dist

```diff
@@ -6,38 +6,30 @@
          executionOrder="random"
          failOnWarning="true"
          failOnRisky="true"
-         verbose="true"
+         cacheDirectory=".phpunit.cache"
 >
 
-    <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[direct]=0" />
-        <env name="COLUMNS" value="100" force="true" />
-    </php>
+  <php>
+    <env name="COLUMNS" value="100" force="true"/>
+  </php>
 
-    <testsuites>
-        <testsuite name="Infection Test Suite">
-            <directory>tests/phpunit</directory>
+  <testsuites>
+    <testsuite name="Infection Test Suite">
+      <directory>tests/phpunit</directory>
+      <exclude>tests/phpunit/AutoReview</exclude>
+      <exclude>tests/phpunit/Fixtures</exclude>
+    </testsuite>
+  </testsuites>
 
-            <exclude>tests/phpunit/AutoReview</exclude>
-            <exclude>tests/phpunit/Fixtures</exclude>
-        </testsuite>
-    </testsuites>
-
-    <coverage cacheDirectory="./reports/.coverage-cache">
```

Comment:
- [ ] check if we still have cache

Reply:
good thing, it's now the only one cache on the top level `<phpunit>` tag: https://docs.phpunit.de/en/10.5/configuration.html#the-cachedirectory-attribute